src/OrchardCore.Modules/OrchardCore.ContentTypes/Views/Admin/EditField.cshtml

```diff
@@ -106,13 +106,15 @@
         $('.field-display').hide();
         $('#field-display-container').show();
         var fieldDisplaySelect = $('#field-display-select');
-        $('.field-display-' + fieldDisplaySelect.val().toLowerCase()).show();         
-
-        fieldDisplaySelect.change(function () {
-            console.log('boom');
-            $('.field-display').hide();
+        if (fieldDisplaySelect.length > 0) {
             $('.field-display-' + fieldDisplaySelect.val().toLowerCase()).show();
-        });
-    });    
+
+            fieldDisplaySelect.change(function () {
+                console.log('boom');
+                $('.field-display').hide();
```

Comment:
lolz. maybe testing code?

Reply:
`boom`: Sounds like @sebastienros 's catch phrase. I saw it too but didn't want to touch it.